### PR TITLE
Update controller_filters to use before_action

### PR DESCRIPTION
**Pull request by thogg4**

### CODE REVIEW

File: lib/spree/chimpy/controller_filters.rb [hunk 1]
```diff
@@ -3,8 +3,8 @@ module ControllerFilters
     extend ActiveSupport::Concern
 
     included do
-      before_filter :set_mailchimp_params
-      before_filter :find_mail_chimp_params, if: :mailchimp_params?
+      before_action :set_mailchimp_params
```
Comment (braidn) [inline lib/spree/chimpy/controller_filters.rb]: @thogg4 thanks for these bumps! You still looking to get this merged into master?

Comment (thogg4) [reply lib/spree/chimpy/controller_filters.rb]: @braidn yep

Comment (braidn) [reply lib/spree/chimpy/controller_filters.rb]: Do you want this just in master or another branch?

Comment (thogg4) [reply lib/spree/chimpy/controller_filters.rb]: @braidn master

Comment (thogg4) [reply lib/spree/chimpy/controller_filters.rb]: thanks @braidn

**Pull request merged**